workflow/scripts/build_egon_data.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# SPDX-FileCopyrightText: : 2024- The PyPSA-Eur Authors
+#
+# SPDX-License-Identifier: MIT
+"""
+Load and prepares the data of the eGo^N DemandRegio project about district
```

Comment:
eGo^N is written here differently than in the other script, check for the right spelling.

Reply:
you can add the link to the dataset like this
`eGo^N DemandRegio project <https://opendata.ffe.de/project/demandregio/>`.